src/client_side.cc

```diff
@@ -1950,8 +1931,19 @@ ConnStateData::clientParseRequests()
         }
     }
 
-    /* XXX where to 'finish' the parsing pass? */
```

Comment:
I removed this stale XXX. It was added in 2006 commit https://github.com/squid-cache/squid/commit/a5baffbadcda654e9cef5781aab97b829d3dcab6, apparently to indicate the lack of parser cleanup code corresponding to the then-added HttpParserInit() call. Since our request parsing code is (or at least pretends to be) incremental, the lack of that cleanup is not really an XXX anymore, and the corresponding parser reset logic is documented in Http::One::Server::parseOneRequest().